frameworks/fmi2api/src/main/java/org/intocps/maestro/framework/fmi2/api/mabl/ExecutionEnvironmentFmi2Api.java

```diff
@@ -57,4 +54,8 @@ public StringVariableFmi2Api getString(String id) {
         Fmi2Builder.Variable<PStm, StringExpressionValue> v = module.call(stringFunc, id);
         return (StringVariableFmi2Api) v;
     }
+
+    public String getEnvName(ComponentVariableFmi2Api comp, PortFmi2Api port) {
+        return String.format("%s_%s_%s", comp.getOwner().getName(), comp.getName(), port.getName());
+    }
```

Comment:
Port actually has a function for this  https://github.com/INTO-CPS-Association/maestro/blob/db6a4f4c1bf67b59908570cbe07d09a7d3ed1f47/frameworks/fmi2api/src/main/java/org/intocps/maestro/framework/fmi2/api/mabl/PortFmi2Api.java#L108-L110